### DISCUSSION TITLE
How to get an ABRP Token?

### DISCUSSION THREAD

**Discussion by thgau** [tillsteinbach/VWsFriend#78] [Q&A] [answered]
How do I get an ABRP Token?
Can I use the browser or the ABRP App or do I have to be in the CarPlay mode?
I tried everywhere but did not find any place with a Token.

Using ID.4 and iPhone

**Comment by tillsteinbach** [comment, marked as answer]
It is a bit tricky. You go in the app to settings and put the slider to detailed settings. Then you click on your car and there you can configure the live settings.
What you are looking for is the generic mode. There it displays the token.
I will try to add some screenshots in the future.

**Comment by woutercoppens** [comment]
Is this still valid? I cannot find a possibility to configure the live settings nor a token.

**Reply by tillsteinbach** [reply]
Yes, at least in the App I still have the settings

**Reply by woutercoppens** [reply]
Ok, I found it. I have to add a new car and there I have the options.
